### ISSUE TITLE
Clicking on Markup does not open the source file

### ISSUE THREAD

**Issue by Malak67** [hjalmers/angular-exemplify#190]
Hello,

We ran into an issue lately where sometimes, we click on the Markup button but nothing happens. After we click on the Show button, it opens the source file, and afterwards the Markup button works as well. 
This happens sometimes, usually when the page first reloads.

We are using the latest version "angular-exemplify": "^3.1.4".
Do you know anything about this issue?

Thanks,

**Comment by hjalmers** [comment]
Hi @Malak67, nothing I've experienced myself I'm afraid. Is the example hidden by default in your app? It sounds like the state is out of sync and clicking "Show" probably solves the state issue so that the toggle works as expected again, if that's the case it should be pretty easy to reproduce but you said it didn't happen all the time right?

**Comment by Malak67** [comment]
Yes, you are correct. It is hidden by default. Another thing I notice is, when hovering on the Markup button, the underline is flickering, like it is loosing the hover event, and when clicking it, nothing happens. After clicking Show, and afterwards hovering on the Markup, it behaves normally. When hovering it shows the underline, but not flickering.

**Comment by hjalmers** [comment]
Hmm, I'll see if I can reproduce it myself. I'm moving this week and I'm going away over the weekend so I probably won't have time to look at it this until next week. You wouldn't happen to have any code or something live where this error occurs?

**Comment by Malak67** [comment]
Sure, next week is fine. Nope, I don`t have any live code. If you don`t manage to reproduce it, I will try testing it in a new project and get back to you.
